apps/frontend/src/pages/profile/components/edit-user-form/edit-user-form.tsx

```diff
@@ -0,0 +1,87 @@
+import { Button, Input } from "~/libs/components/components.js";
+import { areObjectsEqual } from "~/libs/helpers/helpers.js";
+import {
+	useAppDispatch,
+	useAppForm,
+	useCallback,
+	useEffect,
+	useState,
+} from "~/libs/hooks/hooks.js";
+import { actions as authActions } from "~/modules/auth/auth.js";
+import {
+	type UserInfoResponseDto,
+	actions as usersActions,
+} from "~/modules/users/users.js";
+
+import styles from "./styles.module.css";
+
+type Properties = {
+	defaultValues: UserInfoResponseDto;
+	userId: number;
+};
+
+const EditUserForm = ({ defaultValues, userId }: Properties): JSX.Element => {
+	const dispatch = useAppDispatch();
+
+	const { control, errors, handleSubmit, watch } =
+		useAppForm<UserInfoResponseDto>({
+			defaultValues,
+		});
+
+	const [isChanged, setIsChanged] = useState<boolean>(false);
+
+	const watchedValues = watch();
+
+	const handleFormSubmit = useCallback(
+		(event_: React.BaseSyntheticEvent): void => {
```

Comment:
Why there is underscore `_` ?
```suggestion
		(event: React.BaseSyntheticEvent): void => {
```

Reply:
I took the example from sign in/up forms

Reply:
yep, we can't use event because we have an `event` global variable